### PR TITLE
DOCS-8276

### PR DESCRIPTION
**Pull request by rkumar-mongo**
This change is [Reviewable](https://reviewable.io/reviews/mongodb/docs/2730)

### CODE REVIEW

File: source/administration/monitoring.txt [hunk 1]
```diff
@@ -354,6 +354,55 @@ affect logging:
...
 
+.. _monitoring-log-redaction:
+
+Log Redaction
+~~~~~~~~~~~~~
+
+.. versionadded:: 3.4 Available in MongoDB Enterprise only
+
+A :program:`mongod` running with :setting:`security.redactClientLogData`
+removes :doc:`messages </reference/log-messages>` associated with any given log
```
Comment (kay-kim) [inline source/administration/monitoring.txt]: Q: does this remove messages?  as in it first logs and then redacts messages? or is it that the redacted message is logged.

File: source/includes/options-mongod.yaml [hunk 2]
```diff
@@ -2164,4 +2164,37 @@ description: |
...
 optional: true
+---
+program: mongod
+name: redactClientLogData
+args: null
+directive: option
+description: |
+  .. versionadded:: 3.4 Available in MongoDB Enterprise only.
+  
+  A {{program}} running with {{role}} removes any message accompanying a given
```
Comment (kay-kim) [inline source/includes/options-mongod.yaml]: same question with "remove" .

**Comment by rkumar-mongo** [comment]
@kay-kim , pushed fixes plus an additional note. 

Note is based on continuing work to refine the output, so users aren't surprised if output changes at GA.

File: source/core/security-encryption-at-rest.txt [hunk 3]
```diff
@@ -99,6 +99,25 @@ transport encryption.
 
 For details, see :ref:`rotate-encryption-keys`.
 
+Logging
+~~~~~~~
+
+.. versionadded:: 3.4 Available in MongoDB Enterprise only
```
Comment (kay-kim) [inline source/core/security-encryption-at-rest.txt]: Just wondering:  Given that under the Encrypted Storage Engine header we state that the feature is Enterprise only, do we need to reiterate the enterprise availability here as well?

**Comment by kay-kim** [comment]
merged!

**Pull request closed without merging**